app/models/manageiq/providers/vmware/infra_manager/provision/configuration/disk.rb

```diff
@@ -21,6 +21,36 @@ def build_config_disk_spec(vmcs)
     end
   end
 
+  def build_disk_relocate_spec(clone_options)
+    get_disks_info.map do |disk|
+      VimHash.new("VirtualDiskFlatVer2BackingInfo") do |vdbi|
+        case get_option(:disk_format)
+        when 'thin'
+          vdbi.thinProvisioned = true # VimString.new('sparse', "VirtualMachineRelocateTransformation")
+        when 'thick'
+          # is this necessary??
```

Comment:
I think so because the template's disks could be thin and they want the new vm's disks to be thick